installbuilder/datafiles/Base_DSC.data

```diff
@@ -262,6 +261,9 @@ if [ -f /opt/microsoft/dsc/Scripts/InstallModule.py ]; then
   fi
 fi
 
+# Ensure .reg files all have correct permissions
+chmod 644 $OMI_REGISTER_DIR/root-oms/*.reg
```

Comment:
maybe do the same for diy dsc?

Reply:
done

Reply:
Offline - decided not to do this since DIY DSC takes a different code path and we don't have a planned release date for DIY DSC currently.